app/controllers/state_file/questions/submission_pdfs_controller.rb

```diff
@@ -3,10 +3,17 @@ module Questions
     class SubmissionPdfsController < QuestionsController
       def show
         @submission = current_intake.efile_submissions.find_by(id: params[:id])
-        error_redirect and return unless @submission.present?
+        error_redirect and return #unless @submission.present?
```

Comment:
should we take out this comment?

Reply:
omg you're totally right I was just commenting that out for testing 🙈 thanks for catching that @tahsinaislam